packages/upload-api/src/blob/lib.js

```diff
@@ -0,0 +1,85 @@
+import { Failure } from '@ucanto/server'
+
+export const BlobItemNotFoundName = 'BlobItemNotFound'
+export class BlobItemNotFound extends Failure {
+  /**
+   * @param {import('@ucanto/interface').DID} [space]
+   */
+  constructor(space) {
+    super()
+    this.space = space
+  }
+
+  get name() {
+    return BlobItemNotFoundName
+  }
+
+  describe() {
+    if (this.space) {
```

Comment:
I'm not sure why would we capture space given that we either have blob or not and space is irrelevant.